### DISCUSSION TITLE
BEEP sound with Multiband-Melgan

### DISCUSSION THREAD

**Discussion by tekinek** [coqui-ai/TTS#909] [General Q&A]
I am having a trouble with multiband-melgan:

1) As shown in the following figures, there is a consistant "BEEP"  sound in the background of the generated speech. Keep training couldn't get rid of it. The dataset is clean and single-speaker.

<img width="1106" alt="Screen Shot 2021-11-01 at 11 54 00 PM" src="https://user-images.githubusercontent.com/68597606/139713747-09d04b94-eb36-44ce-8997-07e75c7e7cd5.png">
<img width="1062" alt="Screen Shot 2021-11-01 at 11 56 52 PM" src="https://user-images.githubusercontent.com/68597606/139713769-fb63ad9f-501c-43aa-bcda-6b5ffe4c9386.png">

2) I have trained the multiband-melgan in an another repo with the same dataset and audio settings (and same model params to my eyes), whose combination with Coqui's tacotron-DDC results in much better sound quality with very silent backgroud:
https://github.com/TensorSpeech/TensorFlowTTS/blob/34358d82a4c91fd70344872f8ea8a405ea84aedb/examples/multiband_melgan/conf/multiband_melgan.v1.yaml
But it is a tensorflow implentation that comsumes  ~5G GPU memory at inference while Coqui's mb-melgan has much smaller memory footprint and fast inference on GPU.
I hope someone help me to find out what is different in two implementation (or in settings), so that I may tweak Coqui's mb-melgan work as good as tensorflow-tts. In both cases I have used default configs of ljspeech.

Thanks!

**Comment by Amith5970** [comment]
have you resolved it?
